db/utilities_hlv_testing.go

```diff
@@ -48,8 +48,7 @@ func (h *HLVAgent) InsertWithHLV(ctx context.Context, key string) (casOut uint64
 	require.NoError(h.t, err)
 	hlv.CurrentVersionCAS = hlvExpandMacroCASValue
 
-	syncData := &SyncData{HLV: hlv}
-	syncDataBytes, err := base.JSONMarshal(syncData)
+	vvDataBytes, err := base.JSONMarshal(hlv)
 	require.NoError(h.t, err)
```

Comment:
nit:

```suggestion
	vvDataBytes, err := base.MustJSONMarshal(h.t, hlv)
```

I might add something that this is is an SDK level insert, but perhaps this is clear.

Reply:
Good suggestion on MustJSONMarshal.  I think the test description covers the intention, but I can tweak it to make it more clear that "a different HLV-aware peer" is a non-SG peer.